// unified-ui/dist/Blockquote.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Blockquote = undefined;

var _templateObject = _taggedTemplateLiteral(['\n  border-left: 4px solid;\n\n  & > p {\n    font-size: inherit;\n  }\n\n  & > p:first-child {\n    margin-top: 0;\n  }\n\n  & > p:last-child {\n    margin-bottom: 0;\n  }\n\n  ', ';\n'], ['\n  border-left: 4px solid;\n\n  & > p {\n    font-size: inherit;\n  }\n\n  & > p:first-child {\n    margin-top: 0;\n  }\n\n  & > p:last-child {\n    margin-bottom: 0;\n  }\n\n  ', ';\n']);

var _styledComponents = require('styled-components');

var _styledComponents2 = _interopRequireDefault(_styledComponents);

var _styledSystem = require('styled-system');

var _Text = require('./Text');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _taggedTemplateLiteral(strings, raw) { return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } })); }

var Blockquote = exports.Blockquote = (0, _styledComponents2.default)(_Text.Text)(_templateObject, _styledSystem.borderColor);

Blockquote.displayName = 'unified.Blockquote';
Blockquote.defaultProps = {
  as: 'blockquote',
  borderColor: 'grays.2',
  color: 'grays.8',
  p: 0,
  pl: 3,
  my: 4,
  ml: 0,
  mr: 0,
  fontSize: 3
};
