// unified-ui/dist/Button.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Button = undefined;

var _templateObject = _taggedTemplateLiteral(['\n  appearance: none;\n  border: none;\n  outline: none;\n  border-radius: 6px;\n  text-decoration: none;\n\n  ', '\n  ', '\n  ', '\n  ', '\n'], ['\n  appearance: none;\n  border: none;\n  outline: none;\n  border-radius: 6px;\n  text-decoration: none;\n\n  ', '\n  ', '\n  ', '\n  ', '\n']);

var _styledComponents = require('styled-components');

var _styledComponents2 = _interopRequireDefault(_styledComponents);

var _styledSystem = require('styled-system');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _taggedTemplateLiteral(strings, raw) { return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } })); }

var Button = exports.Button = _styledComponents2.default.button(_templateObject, _styledSystem.color, _styledSystem.fontSize, _styledSystem.fontWeight, _styledSystem.space);

Button.displayName = 'unified.Button';
Button.defaultProps = {
  bg: 'blues.5',
  color: 'white',
  py: 2,
  px: 4,
  fontWeight: 'bold',
  fontSize: 2
};
