// unified-ui/dist/List.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.UnorderedList = exports.OrderedList = exports.ListItem = undefined;

var _templateObject = _taggedTemplateLiteral(['\n  color: inherit;\n\n  & *:matches(p) {\n    margin: 0;\n  }\n'], ['\n  color: inherit;\n\n  & *:matches(p) {\n    margin: 0;\n  }\n']),
    _templateObject2 = _taggedTemplateLiteral(['\n  & *:matches(ol, ul) {\n    padding-left: 40px;\n  }\n'], ['\n  & *:matches(ol, ul) {\n    padding-left: 40px;\n  }\n']);

var _styledComponents = require('styled-components');

var _styledComponents2 = _interopRequireDefault(_styledComponents);

var _Text = require('./Text');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _taggedTemplateLiteral(strings, raw) { return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } })); }

var ListItem = exports.ListItem = (0, _styledComponents2.default)(_Text.Text)(_templateObject);

var OrderedList = exports.OrderedList = (0, _styledComponents2.default)(_Text.Text)(_templateObject2);

var UnorderedList = exports.UnorderedList = (0, _styledComponents2.default)(_Text.Text)(_templateObject2);

ListItem.displayName = 'unified.ListItem';
ListItem.defaultProps = {
  as: 'li',
  fontSize: 2,
  lineHeight: 'copy'
};

OrderedList.displayName = 'unified.OrderedList';
OrderedList.defaultProps = {
  as: 'ol',
  pl: 0,
  mt: 1,
  mb: 3
};

UnorderedList.displayName = 'unified.UnorderedList';
UnorderedList.defaultProps = {
  as: 'ul',
  pl: 0,
  mt: 1,
  mb: 3
};
