// unified-ui/dist/mdComponents.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _ = require('.');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
  h1: function h1(props) {
    return _react2.default.createElement(_.Heading, _extends({ as: 'h1', mb: 3, fontSize: 5 }, props));
  },
  h2: function h2(props) {
    return _react2.default.createElement(_.Heading, _extends({ as: 'h2', mt: 4, mb: 3, fontSize: 4 }, props));
  },
  h3: function h3(props) {
    return _react2.default.createElement(_.Heading, _extends({ as: 'h3', mt: 4, mb: 3, fontSize: 3 }, props));
  },
  h4: function h4(props) {
    return _react2.default.createElement(_.Heading, _extends({ as: 'h4', mt: 4, mb: 3, fontSize: 2 }, props));
  },
  h5: function h5(props) {
    return _react2.default.createElement(_.Heading, _extends({ as: 'h5', mt: 4, mb: 3, fontSize: 1 }, props));
  },
  h6: function h6(props) {
    return _react2.default.createElement(_.Heading, _extends({ as: 'h6', mt: 4, mb: 3, fontSize: 1, color: 'grays.8' }, props));
  },
  p: function p(props) {
    return _react2.default.createElement(_.Text, _extends({ mt: 1, mb: 3, fontSize: 2 }, props));
  },

  ol: _.OrderedList,
  ul: _.UnorderedList,
  li: _.ListItem,

  blockquote: _.Blockquote,
  inlineCode: _.Code
};
