// unified-ui/dist/names.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _urls;

function _defineProperty(obj, key, value) { if (key in obj) { Object.defineProperty(obj, key, { value: value, enumerable: true, configurable: true, writable: true }); } else { obj[key] = value; } return obj; }

var base = 'https://github.com/';

var wordMap = exports.wordMap = {
  unified: 'uni|fied',
  'syntax-tree': 'syntax|-tree',
  unist: 'uni|st',
  vfile: 'v|file',
  remark: 're|mark',
  micromark: 'micro|mark',
  mdast: 'md|ast',
  rehype: 're|hype',
  hast: 'h|ast',
  retext: 're|text',
  nlcst: 'nl|cst',
  mdx: 'm|dx'
};

var initialMap = exports.initialMap = {
  unified: 'u|ni',
  unist: 's|t',
  'syntax-tree': 's|t',
  vfile: 'v|f',
  remark: 'r|e',
  micromark: 'm|m',
  mdast: 'm|a',
  rehype: 'r|e',
  hast: 'h|a',
  retext: 'r|e',
  nlcst: 'n|c',
  mdx: 'm|dx'
};

var colors = exports.colors = {
  unified: 'unified',
  unist: 'unified',
  vfile: 'unified',
  remark: 'remark',
  micromark: 'remark',
  mdast: 'remark',
  rehype: 'rehype',
  hast: 'rehype',
  retext: 'retext',
  nlcst: 'retext',
  mdx: 'mdx'
};

var urls = exports.urls = (_urls = {}, _defineProperty(_urls, base + 'unifiedjs/unified', 'unified'), _defineProperty(_urls, base + 'syntax-tree', ['syntax-tree', 'unist']), _defineProperty(_urls, base + 'syntax-tree/unist', 'unist'), _defineProperty(_urls, base + 'vfile/vfile', 'vfile'), _defineProperty(_urls, base + 'remarkjs/remark', 'remark'), _defineProperty(_urls, base + 'remarkjs', 'remark'), _defineProperty(_urls, base + 'micromark/micromark', 'micromark'), _defineProperty(_urls, base + 'micromark', 'micromark'), _defineProperty(_urls, base + 'syntax-tree/mdast', 'mdast'), _defineProperty(_urls, base + 'rehypejs/rehype', 'rehype'), _defineProperty(_urls, base + 'rehypejs', 'rehype'), _defineProperty(_urls, base + 'syntax-tree/hast', 'hast'), _defineProperty(_urls, base + 'retextjs/retext', 'retext'), _defineProperty(_urls, base + 'retextjs', 'retext'), _defineProperty(_urls, base + 'syntax-tree/nlcst', 'nlcst'), _defineProperty(_urls, base + 'mdx-js', 'mdx'), _defineProperty(_urls, base + 'mdx-js/mdx', 'mdx'), _urls);
