// unified-ui/dist/Text.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Code = exports.Text = undefined;

var _templateObject = _taggedTemplateLiteral(['\n  font-family: ', ';\n\n  ', '\n  ', '\n  ', '\n  ', '\n  ', '\n'], ['\n  font-family: ', ';\n\n  ', '\n  ', '\n  ', '\n  ', '\n  ', '\n']),
    _templateObject2 = _taggedTemplateLiteral(['\n  font-family: ', ';\n\n  ', ';\n'], ['\n  font-family: ', ';\n\n  ', ';\n']);

var _styledComponents = require('styled-components');

var _styledComponents2 = _interopRequireDefault(_styledComponents);

var _styledSystem = require('styled-system');

var _theme = require('./theme');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _taggedTemplateLiteral(strings, raw) { return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } })); }

var Text = exports.Text = _styledComponents2.default.p(_templateObject, _theme.fontFamilies.sans, _styledSystem.color, _styledSystem.fontSize, _styledSystem.fontWeight, _styledSystem.lineHeight, _styledSystem.space);

Text.displayName = 'unified.Text';
Text.defaultProps = {
  m: 0,
  fontSize: 3,
  lineHeight: 'copy'
};

var Code = exports.Code = (0, _styledComponents2.default)(Text)(_templateObject2, _theme.fontFamilies.mono, _styledSystem.borderRadius);

Code.displayName = 'unified.Code';
Code.defaultProps = {
  as: 'code',
  borderRadius: 4,
  bg: 'grays.1',
  color: 'grays.9',
  px: 2
};
